xtext/org.icyphy.linguafranca/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -1,9 +1,17 @@
 eclipse.preferences.version=1
 org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.methodParameters=do not generate
```

Comment:
I'm not sure this should be checked in. I keep coming back to this issue of folks pushing random Eclipse configuration data to the repo, which I think is a really bad idea, but I recall there being an unresolved discussion about this with @a-sr about this...